meson_options.txt

```diff
@@ -0,0 +1,67 @@
+# feature options
+option(
+  'service',
+  type : 'boolean',
+  value : 'true',
+  description : 'Enable/Disable background service')
+option(
+  'create',
+  type : 'boolean',
+  value : 'true',
+  description : 'Disable bundle creation and modification commands')
+option(
+  'network',
+  type : 'boolean',
+  value : 'true',
+  description : 'Enable/Disable network update mode')
+option(
+  'streaming',
+  type : 'boolean',
+  value : 'true',
+  description : 'Enable/Disable streaming update mode')
+option(
+  'json',
+  type : 'feature',
+  value : 'enabled',
+  description : 'Enable/Disable JSON support')
+option(
+  'gpt',
+  type : 'feature',
+  value : 'auto',
+  description : 'Enable/Disable GPT support')
+
+
+option(
+  'streaming_user',
+  type : 'string',
+  value : 'nobody',
+  description : 'unpriviledged user for the streaming subprocess')
+
+
+
```

Comment:
Fewer empty lines?

Reply:
done and replaced by separating comments